masterlist.yaml

```
@@ -824,3 +824,27 @@ plugins:
       - name: 'master_index.esp'
         display: '[Master Index Plugin](https://www.nexusmods.com/morrowind/mods/42985/) **OR** [Unofficial Morrowind Official Plugins Patched](https://www.nexusmods.com/morrowind/mods/43931/) (**master-index.esp**)'
         condition: 'not active("MasterIndexRedux.esp")'
+
+  - name: 'The_Vanilla_Quest_Tweaks_RP_Choices_Consequences_Super_Mega_Package_-_Ultimate_Edition.ESP'
+    url: [ 'https://www.nexusmods.com/morrowind/mods/47466' ]
```

The URL is missing a trailing slash, & it would benefit from a name:

```suggestion
    url:
      - link: 'https://www.nexusmods.com/morrowind/mods/47466/'
        name: 'The Vanilla Quest Tweaks RP Choices Consequences Super Mega Package - Ultimate Edition'
```